src/main/scala/tilelink/SRAM.scala

```diff
@@ -23,6 +24,7 @@ class TLRAM(
     atomics: Boolean = false,
     beatBytes: Int = 4,
     ecc: ECCParams = ECCParams(),
+    sramReg: Boolean = false, // drive SRAM data output directly into a register => 1 cycle longer response
```

Comment:
`pipeline` instead of `sramReg`?

Reply:
I'm not sure that name is much better. What is actually happening is that I remove the 'ecc-ok' fast path.

BTW, it has been suggested to me offline that perhaps in the pipelined case we should increase ECC repair latency to 3. Thus, we could take the output data uncorrected from registers. Of course, the valid signal would still need to include a suppression for the correction-required case. Do you think this change is worth it?

Reply:
I think the trade-off is between latency in valid vs. data. The current design has a flop for valid and correction on the data. The proposal have flops for data and detection on valid. This is the same trade-off we make for the old 'fast path'. If it was better there, why is it not also better here?